src/web-socket-handler.ts

```diff
@@ -74,6 +74,57 @@ export class WebSocketHandler implements WebSocketInterface {
         return true;
     }
 
+    public static restartableHandleStandardInput(
+        createWS: (() => Promise<WebSocket>),
+        stdin: stream.Readable | any,
+        streamNum: number = 0,
+        retryCount: number = 3,
+    ): (() => WebSocket | null) {
+        if (retryCount < 0) {
+            throw new Error("retryCount can't be lower than 0.");
+        }
+
+        let queue: Promise<any> = Promise.resolve();
```

Comment:
This is kind of weird, why not just let it be undefined...

Reply:
If queue is undefined `queue = queue.then(() => processData(data));` will raise an error because call `.then` of undefined.